src/extension/utils/createAndOpenConfFile.ts

```diff
@@ -34,7 +34,9 @@ function convertSourceFormDataToConfFileJSON(
   }
 
   if (inputFormData.mainSolidityFile) {
-    config.files.push(inputFormData.mainSolidityFile)
+    config.files.push(
+      `${inputFormData.mainSolidityFile}:${inputFormData.mainContractName}`,
```

Comment:
You did it the right way. But let's take it a step forward.
You probably noticed that currently, there is no inputs validation. However, we may start adding it.
So, I would recommend you do the following:
- See what happens if the `mainContractName` is `null` (or empty).
- By the way, if it's missing, it's doesn't make it wrong. If Solidity filename is  `{CONTRACT_NAME}.sol`, the run command can be `certoraRun {CONTRACT_NAME}.sol --verify {CONTRACT_NAME}:{SPEC_FILE}`.